src/utils/rttc.ts

```diff
@@ -105,3 +98,14 @@ export const checkIfStatement = (context: Context, test: Value) => {
   const node = context.runtime.nodes[0]
   return isBool(test) ? undefined : new TypeError(node, ' as condition', 'boolean', typeOf(test))
 }
+
+export const checkMemberAccess = (context: Context, obj: Value, prop: Value) => {
```

Comment:
Note that `null.prop` is not getting caught here. Not sure if we should pretend that null is an object for Source... If we are saying that null is not an object in Source, we could make isObject return false for null so that an error "expected an object or array, got null instead" here?

Not too sure about this.

Reply:
Made null.prop a type error instead of a property access error.